src/plonemeeting/portal/core/browser/resources/theme/global.scss

```
@@ -1,421 +1,428 @@
-    background: url(../assets/project.jpg);
-    z-index: -1;
-    top: 0;
-    bottom: 0;
-    left: 0;
-    right: 0;
-    opacity: .2;
-    background-size: 20%;
+  content: "";
+  font-size: 100px;
+  position: absolute;
+  top: 0;
```

Stylelint (17.9.0)

[error] Duplicate property "top" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)